backend/bang-ggood/src/main/java/com/bang_ggood/global/exception/ExceptionCode.java

```diff
@@ -1,103 +1,101 @@
 
     // MaintenanceItem
-    MAINTENANCE_ITEM_DUPLICATE(HttpStatus.BAD_REQUEST, "중복된 관리비 항목이 존재합니다."),
-    MAINTENANCE_ITEM_INVALID(HttpStatus.BAD_REQUEST, "유효하지 않은 관리비 항목이 입력되었습니다."),
+    MAINTENANCE_ITEM_DUPLICATE(HttpStatus.BAD_REQUEST, BangggoodCode.CHECKLIST_ERROR, "중복된 관리비 항목이 존재합니다."),
+    MAINTENANCE_ITEM_INVALID(HttpStatus.BAD_REQUEST, BangggoodCode.CHECKLIST_ERROR, "유효하지 않은 관리비 항목이 입력되었습니다."),
 
     //like
-    LIKE_ALREADY_EXISTS(HttpStatus.CONFLICT, "체크리스트가 이미 좋아요 상태입니다"),
-    LIKE_NOT_EXISTS(HttpStatus.BAD_REQUEST, "체크리스트 좋아요가 존재하지 않아 삭제할 수 없습니다."),
+    LIKE_ALREADY_EXISTS(HttpStatus.CONFLICT, BangggoodCode.LIKE_ALREADY_EXISTS, "체크리스트가 이미 좋아요 상태입니다."),
+    LIKE_NOT_EXISTS(HttpStatus.BAD_REQUEST, BangggoodCode.LIKE_NOT_EXISTS, "체크리스트 좋아요가 존재하지 않아 삭제할 수 없습니다."),
 
     // Auth
-    AUTHENTICATION_ACCESS_TOKEN_EMPTY(HttpStatus.UNAUTHORIZED, "액세스 토큰이 존재하지 않습니다. 액세스 토큰을 발급해주세요."),
-    AUTHENTICATION_REFRESH_TOKEN_EMPTY(HttpStatus.UNAUTHORIZED, "리프레시 토큰이 존재하지 않습니다. 다시 로그인해주세요."),
-    AUTHENTICATION_TOKEN_EMPTY(HttpStatus.UNAUTHORIZED, "로그인이 필요한 사용자입니다."),
-    AUTHENTICATION_TOKEN_EXPIRED(HttpStatus.UNAUTHORIZED, "토큰이 만료되었습니다."),
-    AUTHENTICATION_TOKEN_INVALID(HttpStatus.UNAUTHORIZED, "토큰 정보가 올바르지 않습니다."),
-    AUTHENTICATION_TOKEN_NOT_OWNED_BY_USER(HttpStatus.UNAUTHORIZED, "해당 유저의 토큰이 아닙니다"),
-    AUTHENTICATION_TOKEN_USER_MISMATCH(HttpStatus.UNAUTHORIZED, "엑세스 토큰과 리프레시 토큰의 소유자가 다릅니다."),
-    AUTHENTICATION_TOKEN_TYPE_MISMATCH(HttpStatus.BAD_REQUEST, "토큰 타입이 올바르지 않습니다."),
-    OAUTH_TOKEN_INTERNAL_EXCEPTION(HttpStatus.INTERNAL_SERVER_ERROR, "카카오 서버와 통신하는 과정 중 예상치 못한 예외가 발생했습니다."),
-    OAUTH_REDIRECT_URI_MISMATCH(HttpStatus.BAD_REQUEST, "일치하는 Redirect URI가 존재하지 않습니다."),
+    AUTHENTICATION_ACCESS_TOKEN_EMPTY(HttpStatus.UNAUTHORIZED, BangggoodCode.AUTH_ACCESS_TOKEN_EMPTY, "액세스 토큰이 존재하지 않습니다. 액세스 토큰을 발급해주세요."),
+    AUTHENTICATION_REFRESH_TOKEN_EMPTY(HttpStatus.UNAUTHORIZED, BangggoodCode.AUTH_TOKEN_EMPTY, "리프레시 토큰이 존재하지 않습니다. 다시 로그인해주세요."),
+    AUTHENTICATION_TOKEN_EMPTY(HttpStatus.UNAUTHORIZED, BangggoodCode.AUTH_TOKEN_EMPTY, "로그인이 필요한 사용자입니다."),
+    AUTHENTICATION_TOKEN_EXPIRED(HttpStatus.UNAUTHORIZED, BangggoodCode.AUTH_TOKEN_INVALID, "토큰이 만료되었습니다."),
+    AUTHENTICATION_TOKEN_INVALID(HttpStatus.UNAUTHORIZED, BangggoodCode.AUTH_TOKEN_INVALID, "토큰 정보가 올바르지 않습니다."),
+    AUTHENTICATION_TOKEN_NOT_OWNED_BY_USER(HttpStatus.UNAUTHORIZED, BangggoodCode.AUTH_TOKEN_NOT_OWNED_BY_USER, "해당 유저의 토큰이 아닙니다."),
+    AUTHENTICATION_TOKEN_USER_MISMATCH(HttpStatus.UNAUTHORIZED, BangggoodCode.AUTH_TOKEN_USER_MISMATCH, "엑세스 토큰과 리프레시 토큰의 소유자가 다릅니다."),
+    AUTHENTICATION_TOKEN_TYPE_MISMATCH(HttpStatus.BAD_REQUEST, BangggoodCode.AUTH_TOKEN_INVALID, "토큰 타입이 올바르지 않습니다."),
+    OAUTH_TOKEN_INTERNAL_EXCEPTION(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.OAUTH_SERVER_ERROR, "카카오 서버와 통신하는 과정 중 예상치 못한 예외가 발생했습니다."),
+    OAUTH_REDIRECT_URI_MISMATCH(HttpStatus.BAD_REQUEST, BangggoodCode.AUTH_TOKEN_INVALID, "일치하는 Redirect URI가 존재하지 않습니다."),
```

Comment:
```suggestion
    OAUTH_REDIRECT_URI_MISMATCH(HttpStatus.BAD_REQUEST, BangggoodCode.OAUTH_SERVER_ERROR, "일치하는 Redirect URI가 존재하지 않습니다."),
```

![image](https://github.com/user-attachments/assets/251059da-c422-4cad-a8ea-0b3fb008273b)